ros/src/computing/perception/detection/lidar_tracker/packages/lidar_imm_ukf_pda_track/include/ukf.h

```diff
@@ -180,23 +249,36 @@ class UKF
 
   void interaction();
 
+  void predictionSUKF(const double dt);
```

Comment:
The reason for keeping SUKF is that it consistents with the rosparam `use_sukf`.
If rosparam is `use_ukf`, user would be confused since algorithm use ukf in either imm-ukf or sukf.